backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to {oadp-full}
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+= Introduction to {oadp-full}
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
```

Comment:
Why is the [title and metadata repeated](https://github.com/openshift/openshift-docs/pull/64198/files#diff-1ee50b000b47077286254c47ff98fdc2505e4a0b5f2b7c2d4b464d8816860ccfR5-R15)?